### ISSUE TITLE
Understand WOC ATO coverage

### ISSUE THREAD

**Issue by colinmurphy01** [18F/weather.gov#41]

**Comment by mgwalker** [comment]
No, I think this is premature. It doesn't make sense to talk about boundaries until we have something to go inside a boundary. What falls inside the WOC boundary will depend on how we want to work with them. We already know they offer a Drupal-as-a-Service kind of offering, which might end up being perfect, or it may be too locked-down for us to use. But we don't even know for sure if we want to use Drupal. 😄

**Comment by colinmurphy01** [comment]
Roger. Then we can deprioritize, we'll just need to communicate that to Andrew and Beckie so that they're not expecting a meeting over the next 2 weeks or so.

**Comment by colinmurphy01** [comment]
This is actively being worked on now. 

Andrew/Shad are both working on it, feedback from leadership meeting

- Spoke it ITSO, SLA is a little light
- Came up with responsibility matrix for security requirements
- Sent Shad  Greg's cost estimate
- Roles and Responsibilities doc is TBD

**Comment by colinmurphy01** [comment]
This is being managed by Shad.

**Comment by mgwalker** [comment]
@colinmurphy01 Are we effectively good-to-go on this issue? The WOC is handling everything except our app?

**Comment by colinmurphy01** [comment]
Yes we can close this since we are setting up an SDD for the next year.

There is still work to do around this, but it'll be around defining our ATO boundary and needs in general compared to the WOC.
